setup/testing/index.ts

```diff
@@ -27,6 +27,8 @@ import { hexToU8a, u8aToHex } from '@polkadot/util';
 import { AddGraphKeyAction, AddKeyUpdate, ConnectionType, EnvironmentInterface, EnvironmentType, Graph, PrivacyType } from '@dsnp/graph-sdk';
 import { HexString } from '@polkadot/util/types';
 
+const PROVIDER_ACCOUNT_SEED_PHRASE = 'come finish flower cinnamon blame year glad tank domain hunt release fatigue';
```

Comment:
Is this seed phrase for some specific account on Testnet? Or just random?

Reply:
It is random from what I can tell. It was already in the `env.template` https://github.com/AmplicaLabs/reconnection-service/blob/aeb16753eed70504164672e65927d1bdb373661b/env.template#L33

Reply:
For now this is the actual seed phrase but can it be something else like a private key ? wondering how wallets will be integrated by super apps

Reply:
Technically I think they all take any key URI, so it takes a lot of different things: https://polkadot.js.org/docs/api/start/keyring/#adding-accounts